index.bs

```diff
@@ -81,25 +80,20 @@ it is likely that authorization requirements will necessitate it.
 
 *This section is non-normative*
 
-This specification uses the terms "access token", "authorization server", "resource server" (RS),
-"authorization endpoint", "token endpoint", "grant type", "access token request", "access token
-response", and "client" defined by The OAuth 2.0 Authorization Framework
-[[!RFC6749]].
+This specification uses the terms "access token", "resource server" (RS), "grant type", and "client"
```

Comment:
The term "Authorization Server" is used in the next paragraph. "Token Endpoint" is mentioned twice in the "Basic Flow" section. I would leave those terms in place, but the others can go.

Reply:
I left out AS because it links directly to the spec defining it in the next paragraph and totally missed token endpoint, since I was matching with case sensitivity switched on.

I added both back in.